app/models/federated_protocols/saml.rb

```diff
@@ -76,7 +76,8 @@ def current_service_provider
     def ialmax_requested_with_authn_context_comparison?
       return unless (current_service_provider&.ial || 1) > 1
 
-      acr_component_value = Vot::LegacyComponentValues.by_name[request.requested_ial_authn_context]
```

Comment:
what if we added a `.for_name(name)` method? (that still used the constant)?

Reply:
makes sense!